stock_custom_exchange_rate_date/tests/test_stock.py

```diff
@@ -14,7 +14,7 @@ def setUpClass(cls):
         cls.eur = cls.env.ref("base.EUR")
         cls.eur.active = True
         cls.today = fields.Date.context_today(cls.env.user)
-        cls.yesterday = cls.today - timedelta(days=1)
+        cls.yesterday = cls.today - timedelta(days=2)
```

Comment:
This doesn't seem correct. If it's not working, there probably is another issue.

Reply:
Reversed change!